components/next_steps_view/steps/invite_members_step/invite_members_step.tsx

```diff
@@ -278,6 +278,20 @@ class InviteMembersStep extends React.PureComponent<Props, State> {
 
     render(): JSX.Element {
         const linkBtn = this.props.isAdmin ? <UpgradeLink telemetryInfo='click_upgrade_invite_members_step'/> : <NotifyLink/>;
+        const subtitle = this.props?.subscriptionStats?.is_paid_tier === 'true' ? (
+            <FormattedMessage
+                id='next_steps_view.invite_members_step.youCanInvite'
+                defaultMessage='You can invite team members using a space or comma between addresses'
+            />
+        ) : (
```

Comment:
@AGMETEOR just realized here that we need to default to this message if they are not in cloud - which I believe would cause `subscriptionStats` to be null or default paid tier to false?

Reply:
Yes, for non-cloud, `subscriptionStats` is null. So the second part of the ternary would be shown